worker/charmcraft.yaml

```diff
@@ -40,8 +43,11 @@ containers:
   tempo:
     resource: tempo-image
     mounts:
-      - storage: data
-        location: /data
+      - storage: config
```

Comment:
I don't think we need to store the configfile, as we can regenerate it any time (and we do: reconciler) from relation data, secrets contents, etc...
@michaeldmitry why did you say we needed this?